source/tutorials/install-nix.rst

```diff
@@ -25,3 +25,20 @@ Check that the installation was successful:
    nix-env (Nix) 2.3.6
 
 .. _verify the installation script: https://nixos.org/download.html#nix-verify-installation
+
+Use Docker to explore Nix
+-------------------------
+
+Start a Docker shell with Nix:
+
+.. code:: bash
+
+    $ docker run -it nixos/nix
+
+Or start a Docker shell with Nix exposing a work directory:
+
+.. code:: bash
+
+    $ mkdir workdir
+    $ docker run -it -v $(pwd)/workdir:/root nixos/nix
```

Comment:
Any particular reason not to mount it at /workdir? To me /root feels abusing the convention of what should be there

Reply:
no reason, moved over to `/workdir`